nipype/interfaces/fsl/tests/test_model.py

```diff
@@ -63,6 +64,4 @@ def test_MultipleRegressDesign(tmpdir):
 1
 """
     for ii in ["mat", "con", "fts", "grp"]:
-        assert tmpdir.join("design." + ii).read() == eval(
-            "design_" + ii + "_expected_content"
-        )
+        assert Path(eval('res.outputs.design_'+ii)).read_text() in eval( "design_" + ii + "_expected_content")
```

Comment:
Then down here we'd do:

```suggestion
        outfile = "design_" + ii
        assert Path(outputs[outfile]).read_text() == expected_content[outfile]
```